utils/output_handling.py

```diff
@@ -73,6 +72,8 @@ def upload_csv(csv_file_path, bigquery_cfg):
     bigquery_cfg: The string representing the BigQuery table config. Comes in
       the form <dataset_id>:<table_id>:<location>
   """
+  from google.cloud import bigquery
```

Comment:
What's the motivation to move this import down here?

Reply:
It's a workaround of https://github.com/bazelbuild/rules_python/issues/14 for `bazel test` to work.